Makeconf

```diff
@@ -1,30 +1,39 @@
-####======= Common Declarations for *all* ESS -*-Makefile-*-s ==========
-
 ## To install ESS for all users on your unix system:
-## 1. Edit Section 1 and 2 below
-## 2. Execute: gmake install
+## 1. Check variables below, edit as necessary
+## 2. make
+## 3. make install
 
 ## Section 1
-## Installation variables for your emacs variant
+## Installation variables for your Emacs variant
 ##
 ## Variable        Description
 ## EMACS           Path to GNU Emacs
-## EMACSBATCH      Emacs command for compiling elisp files
-## SITELISP        Destination of site-lisp; but some variants have no site-lisp
-## LISPDIR         Destination of compiled elisp files
-## INFODIR         Destination of info files
-## ETCDIR          Destination of script and icon files
-## PREFIX(DESTDIR) Directory prepended to SITELISP, INFODIR, DOCDIR & ETCDIR
-##                 Specify either PREFIX or DESTDIR to over-ride /usr
-DESTDIR=/usr
-PREFIX=$(DESTDIR)
+## EMACSBATCH      How to run Emacs as batch
+## SITELISP        Destination of site-lisp
+## ESSDIR          Destination of ESS
+## JULIA_DIR       Path to julia-mode
+DESTDIR ?= /usr
+PREFIX ?= $(DESTDIR)
 
 ##__ GNU Emacs __
 EMACS ?= emacs
-SITELISP=$(PREFIX)/share/emacs/site-lisp
-LISPDIR=$(SITELISP)/ess
-INFODIR=$(PREFIX)/share/info
-ETCDIR =$(PREFIX)/share/emacs/etc/ess
+SITELISP ?= $(PREFIX)/share/emacs/site-lisp
+ESSDIR ?= $(SITELISP)/ess
```

Comment:
Can we move this to makefile proper not in makeconf? Would be nice to get rid of makeconf at some point.

Reply:
They're variables that users/distros can override, and we also need them in other Makefiles (e.g. lisp/Makefile, doc/Makefile). We can't include the top-level makefile in those sub-makefiles. What do you suggest, exactly?

Reply:
> We can't include the top-level makefile

Right. Forgot about those.